README.fr.md

```diff
@@ -0,0 +1,42 @@
+## Liste noire de serveurs
+
+Ce dépôt contient une `blacklist.txt` (liste noire) de domaines XMPP qui sont
+utilisés par des spammeurs et qui ne répondent pas aux plaintes d'abus. Ces
```

Comment:
Always use an apostrophe (U+2019, “’”) instead of a single quote, in French as well as in English, here and below.